### DISCUSSION TITLE
Mixing time suggestions?

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#769] [REM] [answered]
18000 gallons.

On the Intellichem, I kept it low since I couldn't manually dose until the mixing time expired (after the pool pump started running).

I'm not sure REM has the limitation, but generally, what's the down side of having mixing time too low?  Too high?

I'm thinking if mixing time is too low, then the readings aren't accurate, since a previous dose hasn't mixed.  So, an "overshoot" might occur--more acid or chlorine dosed than was needed to reach the target setting.  But, maximum volume per dose would mitigate that.

Mixing time too long?  The ph/ORP could drift more than would occur if the dosing occured sooner.

Either way, it doesn't seem to matter much.  

What are others using?

**Comment by johnny2678** [comment, marked as answer]
10 min mixing time for my 14500 gal 

MA is generally pretty quick to mix in my pool and I can observe dosing effects on the pH readings within that time.  Using 20 Baume MA diluted 1:1 with water if it matters.

Here's yesterday's graph... Didn't top off my tank before leaving for vacation so pH drifted higher until I put more MA in the tank yesterday.  After 8 or 9 dosing cycles, it was right back to 7.8 where it likes to be.

![image](https://github.com/tagyoureit/nodejs-poolController/assets/8219743/4990f5f0-ba58-4829-a9c6-35b45266ad0f)
